tasks/main.yml

```diff
@@ -15,3 +15,4 @@
   when: ansible_distribution == "Ubuntu"
   tags:
     - osquery
+
```

Comment:
Or better yet, remove the file from the commit since nothing really needs to be changed in it.